editor/editor_inspector.cpp

```diff
@@ -37,6 +37,7 @@
 #include "editor/editor_feature_profile.h"
 #include "editor/editor_main_screen.h"
 #include "editor/editor_node.h"
+#include "editor/editor_paths.h"
```

Comment:
What's this for?

Reply:
Blame my IDE for loving to automatically place includes everywhere.